src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,63 @@
 import React from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  setMovies: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovies }) => {
+  const [query, setQuery] = React.useState('');
+  const [movie, setMovie] = React.useState<Movie | null>(null);
+  const [isLoading, setLoading] = React.useState(false);
+  const [isError, setError] = React.useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`